dtApi/src/app/app.module.ts

```diff
@@ -2,12 +2,23 @@ import { BrowserModule } from '@angular/platform-browser'
 import { NgModule } from '@angular/core'
 
 import { AppRoutingModule } from './app-routing.module'
+import { HttpClientModule } from '@angular/common/http'
+import { ReactiveFormsModule } from '@angular/forms'
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations'
```

Comment:
minor. try to group imports (angular, 3rd-party, your files) with empty line after each group